app/controllers/catalog_controller.rb

```diff
@@ -174,4 +175,19 @@ def lookup_manifest
       render json: { message: "No manifest found for #{ark}" }, status: 404
     end
   end
+
+  def numismatics
```

Comment:
Probable better to move this to it's own controller. The FixityDashboardController is an example of this:
https://github.com/pulibrary/figgy/blob/master/app/controllers/fixity_dashboard_controller.rb

Reply:
I suggested yesterday to move it to its own controller when we will refactor numismatics.

Reply:
the requested change was to be a https://figgy.princeton.edu/numismatics. 
If I move it to a different controller would that be:
https://figgy.princeton.edu/numismatics/home ?
https://figgy.princeton.edu/dashboard/numismatics?
or something else?

Reply:
we decided dashboard/numismatics.